osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -101,11 +101,12 @@ def extract_excerpts(execution_configuration, extraction_order, bbox_args, conve
                 .format(bbox_args=bbox_args, format=export_format_key)
             subprocess.check_call(extraction_command.split(' '))
 
-            if len(os.listdir(settings.RESULT_MEDIA_ROOT)) > 0:
-                for result_file_name in os.listdir(settings.RESULT_MEDIA_ROOT):
+            directory = os.path.join(settings.RESULT_MEDIA_ROOT, str(extraction_order.id) + '_' + NAME)
```

Comment:
Rename this local variable, so the name tells the reader what directory this is or what its purpose is. (Renaming only the variable is sufficient. We don't have to rename the argument in `create_output_file`'s signature, as there, `directory`'s purpose is clear from the context given by the function name.)